app/assets/javascripts/privileges.js

```diff
@@ -25,15 +26,18 @@ $(() => {
     const $td = $tgt.parent();
     const $input = $td.find('.js-privilege-edit');
     const name = $input.data('name');
-    const value = $input.val();
+    const type = $input.data('type');
+    let value = $input.val();
+
+    const value = parseFloat($input.val() || '') || null;
```

Comment:
Redeclaration of `value`, which prevents compilation. Removing line 30 should solve this.